[2025-09-03T04:40:13.852+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: institution_detail_data_quality_pipeline.validate manual__2025-09-03T04:40:07.940319+00:00 [queued]>
[2025-09-03T04:40:13.857+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: institution_detail_data_quality_pipeline.validate manual__2025-09-03T04:40:07.940319+00:00 [queued]>
[2025-09-03T04:40:13.857+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2025-09-03T04:40:13.857+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2025-09-03T04:40:13.857+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2025-09-03T04:40:13.866+0000] {taskinstance.py:1300} INFO - Executing <Task(BranchPythonOperator): validate> on 2025-09-03 04:40:07.940319+00:00
[2025-09-03T04:40:13.869+0000] {standard_task_runner.py:55} INFO - Started process 1933 to run task
[2025-09-03T04:40:13.870+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'institution_detail_data_quality_pipeline', 'validate', 'manual__2025-09-03T04:40:07.940319+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/dag_dataquality.py', '--cfg-path', '/tmp/tmpb5qorgnn']
[2025-09-03T04:40:13.871+0000] {standard_task_runner.py:83} INFO - Job 11: Subtask validate
[2025-09-03T04:40:13.882+0000] {logging_mixin.py:137} WARNING - /usr/local/lib/python3.9/site-packages/***/settings.py:249 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-09-03T04:40:13.911+0000] {task_command.py:388} INFO - Running <TaskInstance: institution_detail_data_quality_pipeline.validate manual__2025-09-03T04:40:07.940319+00:00 [running]> on host 56e79e5a4e8c
[2025-09-03T04:40:13.952+0000] {taskinstance.py:1507} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Klaus_Rezende
AIRFLOW_CTX_DAG_ID=institution_detail_data_quality_pipeline
AIRFLOW_CTX_TASK_ID=validate
AIRFLOW_CTX_EXECUTION_DATE=2025-09-03T04:40:07.940319+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-09-03T04:40:07.940319+00:00
[2025-09-03T04:40:13.953+0000] {logging_mixin.py:137} INFO - ðŸ“Š Data Quality Results: 80.0%
[2025-09-03T04:40:13.953+0000] {python.py:177} INFO - Done. Returned value was: invalid
[2025-09-03T04:40:13.953+0000] {python.py:211} INFO - Branch callable return invalid
[2025-09-03T04:40:13.953+0000] {skipmixin.py:155} INFO - Following branch invalid
[2025-09-03T04:40:13.953+0000] {skipmixin.py:211} INFO - Skipping tasks ['valid']
[2025-09-03T04:40:13.988+0000] {taskinstance.py:1318} INFO - Marking task as SUCCESS. dag_id=institution_detail_data_quality_pipeline, task_id=validate, execution_date=20250903T044007, start_date=20250903T044013, end_date=20250903T044013
[2025-09-03T04:40:14.002+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2025-09-03T04:40:14.019+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
